test/specs/util/xhr.js

```
@@ -40,7 +41,7 @@ define(['util/xhr', 'nbd/Promise'], function(xhr, Promise) {
 
       response.then(success, error).then(function() {
         expect(success).toHaveBeenCalled();
-        expect(succes).toHaveBeenCalledWith(text);
+        expect(success).toHaveBeenCalledWith(text);
         expect(error).not.toHaveBeenCalled();
       }).finally(done);
```

OH. finally done! Swallow that error friend.